app/printfile_sorting.py

```diff
@@ -0,0 +1,55 @@
+import shutil
+from pathlib import Path
+
+from csvsort import csvsort
+
+from app.constants import PackCode, ActionType, PrintFileSorting
+from app.mappings import ACTION_TYPE_TO_PRINT_TEMPLATE
+from config import Config
+
+
+def sort_print_file_if_required(complete_partial_file: Path, pack_code: PackCode, action_type: ActionType,
+                                context_logger):
+    if pack_code in PrintFileSorting.PACKCODES_TO_SORT:
+        file_to_sort = copy_file_to_sorting_dir_to_sort(complete_partial_file)
+        print_template = ACTION_TYPE_TO_PRINT_TEMPLATE.get(action_type)
```

Comment:
Might as well be
```suggestion
        print_template = ACTION_TYPE_TO_PRINT_TEMPLATE[action_type]
```
We'd want this to fall over early it couldn't map to a print template rather than passing `none` through